cap-ci/scripts/deploy_k8s.tmpl

```diff
@@ -1,38 +1,39 @@
 {{ define "scripts_deploy_k8s" }}
 {{- /* create k8s cluster */}}
 export random_variable=$(hexdump -n 8 -e '2/4 "%08x"' /dev/urandom)
-export AWS_ACCESS_KEY_ID='((aws-ci-chuller-access-key-id))'
-export AWS_SECRET_ACCESS_KEY='((aws-ci-chuller-secret-access-key))'
+export CLUSTER_NAME=ci-${BACKEND}-${random_variable}
+export TF_KEY=${CLUSTER_NAME}
+{{- /* required for eks and storing tf state in s3 backend */}}
+export AWS_ACCESS_KEY_ID='((aws-service-account-ci-creds.access-key))'
+export AWS_SECRET_ACCESS_KEY='((aws-service-account-ci-creds.secret-key))'
+{{- /* adding ssh key for the ci */}}
+printf "%s" '((ssh-key-ci.pvt-key))' > $PWD/sshkey
+printf "%s" '((ssh-key-ci.pub-key))' > $PWD/sshkey.pub
+chmod 600 $PWD/sshkey
+chmod 600 $PWD/sshkey.pub
+eval $(ssh-agent -s)
+ssh-add $PWD/sshkey
 if [[ ${BACKEND} == "gke" ]]; then
   printf "%s" '((gke-key-json))' > $PWD/gke-key.json
   export GKE_CRED_JSON=$PWD/gke-key.json
-  export GKE_CLUSTER_NAME=ci-${BACKEND}-${random_variable}
```

Comment:
Why not export the variables? :thinking: 
I would like a solution that allows me to hijack the container, and have all the variables set up directly in the shell.